### ISSUE TITLE
Does not launch (SIGSERV)

### ISSUE THREAD

**Issue by cosad3s** [Bearer/bearer#910]
## Description & Reproduction

**Installed with:**

```bash
curl -sfL https://raw.githubusercontent.com/Bearer/bearer/main/contrib/install.sh | sh
```

**Launch:**

```bash
❯ bearer scan myrepo
Loading rules
fatal error: unexpected signal during runtime execution
[signal SIGSEGV: segmentation violation code=0x1 addr=0x1d3560 pc=0x7f61d80372b2]

runtime stack:
runtime.throw({0x15499fd?, 0x353?})
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/panic.go:1047 +0x5d fp=0x7ffea3dffff8 sp=0x7ffea3dfffc8 pc=0x43c57d
runtime.sigpanic()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/signal_unix.go:819 +0x369 fp=0x7ffea3e00048 sp=0x7ffea3dffff8 pc=0x4521a9

goroutine 26 [syscall]:
runtime.cgocall(0xeec6b0, 0xc00005fd98)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/cgocall.go:158 +0x5c fp=0xc00005fd70 sp=0xc00005fd38 pc=0x408b3c
net._C2func_getaddrinfo(0xc0011834c0, 0x0, 0xc00119c750, 0xc000119a68)
	_cgo_gotypes.go:94 +0x56 fp=0xc00005fd98 sp=0xc00005fd70 pc=0x61bf16
net.cgoLookupIPCNAME.func1({0xc0011834c0, 0x0?, 0x0?}, 0xc00014bba8?, 0x620cfb?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/cgo_unix.go:160 +0x9f fp=0xc00005fdf0 sp=0xc00005fd98 pc=0x61d7bf
net.cgoLookupIPCNAME({0x13c633c, 0x3}, {0xc00014bba8, 0xe})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/cgo_unix.go:160 +0x173 fp=0xc00005ff38 sp=0xc00005fdf0 pc=0x61d013
net.cgoIPLookup(0x3f7b150?, {0x13c633c?, 0xc00014bbc0?}, {0xc00014bba8?, 0xc000160dc0?})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/cgo_unix.go:217 +0x3b fp=0xc00005ffa8 sp=0xc00005ff38 pc=0x61d87b
net.cgoLookupIP.func1()
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/cgo_unix.go:227 +0x36 fp=0xc00005ffe0 sp=0xc00005ffa8 pc=0x61dcb6
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc00005ffe8 sp=0xc00005ffe0 pc=0x46cee1
created by net.cgoLookupIP
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/cgo_unix.go:227 +0x12a

goroutine 1 [select]:
runtime.gopark(0xc000f1e7d8?, 0x4?, 0x18?, 0x0?, 0xc000f1e680?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000f1e4d8 sp=0xc000f1e4b8 pc=0x43f196
runtime.selectgo(0xc000f1e7d8, 0xc000f1e678, 0xc001194ee0?, 0x0, 0x0?, 0x1)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/select.go:328 +0x7bc fp=0xc000f1e618 sp=0xc000f1e4d8 pc=0x44e69c
net/http.(*Transport).getConn(0x3be0ca0, 0xc0011840c0, {{}, 0x0, {0xc0011506c0, 0x5}, {0xc00014bba8, 0x12}, 0x0})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/transport.go:1376 +0x5c6 fp=0xc000f1e830 sp=0xc000f1e618 pc=0x711b26
net/http.(*Transport).roundTrip(0x3be0ca0, 0xc001180200)
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/transport.go:582 +0x75e fp=0xc000f1ea58 sp=0xc000f1e830 pc=0x70d7de
net/http.(*Transport).RoundTrip(0x203000?, 0x19af520?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/roundtrip.go:17 +0x19 fp=0xc000f1ea78 sp=0xc000f1ea58 pc=0x6f5579
net/http.send(0xc001180200, {0x19af520, 0x3be0ca0}, {0x139ca00?, 0x1?, 0x0?})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/client.go:251 +0x5f7 fp=0xc000f1ec70 sp=0xc000f1ea78 pc=0x6b6c77
net/http.(*Client).send(0x3f7bb20, 0xc001180200, {0xc000409cb0?, 0x30?, 0x0?})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/client.go:175 +0x9b fp=0xc000f1ece8 sp=0xc000f1ec70 pc=0x6b64fb
net/http.(*Client).do(0x3f7bb20, 0xc001180200)
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/client.go:715 +0x8fc fp=0xc000f1eed8 sp=0xc000f1ece8 pc=0x6b887c
net/http.(*Client).Do(...)
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/client.go:581
github.com/google/go-github/github.(*Client).Do(0xc001159cc0, {0x19b6bf8, 0xc000124000}, 0xc001180100, {0x1387920, 0xc001161f40})
	/home/runner/go/pkg/mod/github.com/google/go-github@v17.0.0+incompatible/github/github.go:473 +0x255 fp=0xc000f1eff0 sp=0xc000f1eed8 pc=0x729a75
github.com/google/go-github/github.(*RepositoriesService).getSingleRelease(0xc001159d48, {0x19b6bf8, 0xc000124000}, {0xc000045890?, 0x2?})
	/home/runner/go/pkg/mod/github.com/google/go-github@v17.0.0+incompatible/github/repos_releases.go:121 +0x8d fp=0xc000f1f048 sp=0xc000f1eff0 pc=0x72b72d
github.com/google/go-github/github.(*RepositoriesService).GetLatestRelease(0xc00011bb60?, {0x19b6bf8, 0xc000124000}, {0x13e3373?, 0xc0001141e0?}, {0x13e3373, 0x6})
	/home/runner/go/pkg/mod/github.com/google/go-github@v17.0.0+incompatible/github/repos_releases.go:103 +0xc5 fp=0xc000f1f0a8 sp=0xc000f1f048 pc=0x72b645
github.com/bearer/bearer/pkg/commands/artifact.Run({_, _}, {{{0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}}, {{0x0, 0x0}, ...}, ...}, ...)
	/home/runner/work/bearer/bearer/pkg/commands/artifact/run.go:223 +0xbc fp=0xc000f1f8a0 sp=0xc000f1f0a8 pc=0xdfacdc
github.com/bearer/bearer/pkg/commands.NewScanCommand.func2(0xc001162900, {0xc001131490?, 0x1, 0x1})
	/home/runner/work/bearer/bearer/pkg/commands/scan.go:93 +0x4a5 fp=0xc000f1fd00 sp=0xc000f1f8a0 pc=0xeebcc5
github.com/spf13/cobra.(*Command).execute(0xc001162900, {0xc001131450, 0x1, 0x1})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x862 fp=0xc000f1fe38 sp=0xc000f1fd00 pc=0x8f92c2
github.com/spf13/cobra.(*Command).ExecuteC(0xc001162000)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd fp=0xc000f1fef0 sp=0xc000f1fe38 pc=0x8f9b3d
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
main.run()
	/home/runner/work/bearer/bearer/cmd/bearer/main.go:21 +0x3a fp=0xc000f1ff20 sp=0xc000f1fef0 pc=0xeec5ba
main.main()
	/home/runner/work/bearer/bearer/cmd/bearer/main.go:13 +0x1d fp=0xc000f1ff80 sp=0xc000f1ff20 pc=0xeec4dd
runtime.main()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:250 +0x212 fp=0xc000f1ffe0 sp=0xc000f1ff80 pc=0x43edd2
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000f1ffe8 sp=0xc000f1ffe0 pc=0x46cee1

goroutine 2 [force gc (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000062fb0 sp=0xc000062f90 pc=0x43f196
runtime.goparkunlock(...)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:369
runtime.forcegchelper()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:302 +0xad fp=0xc000062fe0 sp=0xc000062fb0 pc=0x43f02d
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000062fe8 sp=0xc000062fe0 pc=0x46cee1
created by runtime.init.6
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:290 +0x25

goroutine 3 [GC sweep wait]:
runtime.gopark(0x1?, 0x0?, 0x0?, 0x0?, 0x0?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000063790 sp=0xc000063770 pc=0x43f196
runtime.goparkunlock(...)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:369
runtime.bgsweep(0x0?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgcsweep.go:297 +0xd7 fp=0xc0000637c8 sp=0xc000063790 pc=0x429cf7
runtime.gcenable.func1()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:178 +0x26 fp=0xc0000637e0 sp=0xc0000637c8 pc=0x41e966
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000637e8 sp=0xc0000637e0 pc=0x46cee1
created by runtime.gcenable
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:178 +0x6b

goroutine 4 [GC scavenge wait]:
runtime.gopark(0xc00007a000?, 0x19aa8c0?, 0x0?, 0x0?, 0x0?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000063f70 sp=0xc000063f50 pc=0x43f196
runtime.goparkunlock(...)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:369
runtime.(*scavengerState).park(0x3f7d740)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgcscavenge.go:389 +0x53 fp=0xc000063fa0 sp=0xc000063f70 pc=0x427d53
runtime.bgscavenge(0x0?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgcscavenge.go:622 +0x65 fp=0xc000063fc8 sp=0xc000063fa0 pc=0x428345
runtime.gcenable.func2()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:179 +0x26 fp=0xc000063fe0 sp=0xc000063fc8 pc=0x41e906
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000063fe8 sp=0xc000063fe0 pc=0x46cee1
created by runtime.gcenable
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:179 +0xaa

goroutine 18 [finalizer wait]:
runtime.gopark(0x3f7dfe0?, 0xc0001041a0?, 0x0?, 0x0?, 0xc000062770?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000062628 sp=0xc000062608 pc=0x43f196
runtime.goparkunlock(...)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:369
runtime.runfinq()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mfinal.go:180 +0x10f fp=0xc0000627e0 sp=0xc000062628 pc=0x41da6f
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0000627e8 sp=0xc0000627e0 pc=0x46cee1
created by runtime.createfing
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mfinal.go:157 +0x45

goroutine 19 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc00005e750 sp=0xc00005e730 pc=0x43f196
runtime.gcBgMarkWorker()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc00005e7e0 sp=0xc00005e750 pc=0x420ab1
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc00005e7e8 sp=0xc00005e7e0 pc=0x46cee1
created by runtime.gcBgMarkStartWorkers
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:1159 +0x25

goroutine 34 [GC worker (idle)]:
runtime.gopark(0x1176828c4ed8?, 0x0?, 0x0?, 0x0?, 0x0?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000508750 sp=0xc000508730 pc=0x43f196
runtime.gcBgMarkWorker()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc0005087e0 sp=0xc000508750 pc=0x420ab1
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0005087e8 sp=0xc0005087e0 pc=0x46cee1
created by runtime.gcBgMarkStartWorkers
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:1159 +0x25

goroutine 35 [GC worker (idle)]:
runtime.gopark(0x117682a05353?, 0x3?, 0x81?, 0xe5?, 0x0?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000508f50 sp=0xc000508f30 pc=0x43f196
runtime.gcBgMarkWorker()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc000508fe0 sp=0xc000508f50 pc=0x420ab1
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000508fe8 sp=0xc000508fe0 pc=0x46cee1
created by runtime.gcBgMarkStartWorkers
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:1159 +0x25

goroutine 36 [GC worker (idle)]:
runtime.gopark(0x117682a00ea7?, 0x0?, 0x0?, 0x0?, 0x0?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000509750 sp=0xc000509730 pc=0x43f196
runtime.gcBgMarkWorker()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:1235 +0xf1 fp=0xc0005097e0 sp=0xc000509750 pc=0x420ab1
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0005097e8 sp=0xc0005097e0 pc=0x46cee1
created by runtime.gcBgMarkStartWorkers
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/mgc.go:1159 +0x25

goroutine 24 [select]:
runtime.gopark(0xc0001a14d8?, 0x2?, 0x9f?, 0x24?, 0xc0001a141c?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc0001a1258 sp=0xc0001a1238 pc=0x43f196
runtime.selectgo(0xc0001a14d8, 0xc0001a1418, 0x12?, 0x0, 0x0?, 0x1)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/select.go:328 +0x7bc fp=0xc0001a1398 sp=0xc0001a1258 pc=0x44e69c
net.(*Resolver).lookupIPAddr(0x3f7b140, {0x19b6c30?, 0xc00011bda0}, {0x13c633c, 0x3}, {0xc00014bba8, 0xe})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/lookup.go:327 +0x51b fp=0xc0001a15a8 sp=0xc0001a1398 pc=0x60a45b
net.(*Resolver).internetAddrList(0x19b6c30?, {0x19b6c30?, 0xc00011bda0?}, {0x13c633c, 0x3}, {0xc00014bba8?, 0x12?})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/ipsock.go:288 +0x67a fp=0xc0001a16c8 sp=0xc0001a15a8 pc=0x6081ba
net.(*Resolver).resolveAddrList(0x3fbdf48?, {0x19b6c30, 0xc00011bda0}, {0x13cca68, 0x4}, {0x13c633c?, 0x436190?}, {0xc00014bba8, 0x12}, {0x0, ...})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/dial.go:222 +0x41b fp=0xc0001a17d0 sp=0xc0001a16c8 pc=0x5f247b
net.(*Dialer).DialContext(0xc00011a180, {0x19b6bf8, 0xc000124000}, {0x13c633c, 0x3}, {0xc00014bba8, 0x12})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/dial.go:410 +0x448 fp=0xc0001a1920 sp=0xc0001a17d0 pc=0x5f3428
net.(*Dialer).DialContext-fm({0x19b6bf8?, 0xc000124000?}, {0x13c633c?, 0x60?}, {0xc00014bba8?, 0x4127e7?})
	<autogenerated>:1 +0x5b fp=0xc0001a1968 sp=0xc0001a1920 pc=0x72773b
net/http.(*Transport).dial(0x1000041249f?, {0x19b6bf8?, 0xc000124000?}, {0x13c633c?, 0x0?}, {0xc00014bba8?, 0xc000070a68?})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/transport.go:1170 +0xda fp=0xc0001a19c0 sp=0xc0001a1968 pc=0x710e5a
net/http.(*Transport).dialConn(0x3be0ca0, {0x19b6bf8, 0xc000124000}, {{}, 0x0, {0xc0011506c0, 0x5}, {0xc00014bba8, 0x12}, 0x0})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/transport.go:1608 +0x83f fp=0xc0001a1ed8 sp=0xc0001a19c0 pc=0x713f1f
net/http.(*Transport).dialConnFor(0x0?, 0xc000160dc0)
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/transport.go:1450 +0xb0 fp=0xc0001a1fc0 sp=0xc0001a1ed8 pc=0x7127b0
net/http.(*Transport).queueForDial.func1()
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/transport.go:1419 +0x2a fp=0xc0001a1fe0 sp=0xc0001a1fc0 pc=0x7126ca
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc0001a1fe8 sp=0xc0001a1fe0 pc=0x46cee1
created by net/http.(*Transport).queueForDial
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/http/transport.go:1419 +0x3f2

goroutine 25 [select]:
runtime.gopark(0xc000070c28?, 0x2?, 0x0?, 0x30?, 0xc000070c0c?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/proc.go:363 +0xd6 fp=0xc000070a98 sp=0xc000070a78 pc=0x43f196
runtime.selectgo(0xc000070c28, 0xc000070c08, 0xc0011833ca?, 0x0, 0xc001159ea0?, 0x1)
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/select.go:328 +0x7bc fp=0xc000070bd8 sp=0xc000070a98 pc=0x44e69c
net.cgoLookupIP({0x19b6bc0, 0xc001184140}, {0x13c633c, 0x3}, {0xc00014bba8, 0xe})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/cgo_unix.go:228 +0x1aa fp=0xc000070cc8 sp=0xc000070bd8 pc=0x61daca
net.(*Resolver).lookupIP(0x3f7b140, {0x19b6bc0, 0xc001184140}, {0x13c633c, 0x3}, {0xc00014bba8, 0xe})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/lookup_unix.go:74 +0x128 fp=0xc000070e18 sp=0xc000070cc8 pc=0x60c1c8
net.(*Resolver).lookupIP-fm({0x19b6bc0?, 0xc001184140?}, {0x13c633c?, 0x0?}, {0xc00014bba8?, 0x0?})
	<autogenerated>:1 +0x5b fp=0xc000070e60 sp=0xc000070e18 pc=0x620cfb
net.glob..func1({0x19b6bc0?, 0xc001184140?}, 0x0?, {0x13c633c?, 0x0?}, {0xc00014bba8?, 0x7127b0?})
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/hook.go:23 +0x3d fp=0xc000070ea0 sp=0xc000070e60 pc=0x5ffc9d
net.(*Resolver).lookupIPAddr.func1()
	/opt/hostedtoolcache/go/1.19.8/x64/src/net/lookup.go:321 +0x9f fp=0xc000070f38 sp=0xc000070ea0 pc=0x60acff
internal/singleflight.(*Group).doCall(0x3f7b150, 0xc001146af0, {0xc00014bbc0, 0x12}, 0xc000160dc0?)
	/opt/hostedtoolcache/go/1.19.8/x64/src/internal/singleflight/singleflight.go:95 +0x3b fp=0xc000070fa8 sp=0xc000070f38 pc=0x5ee71b
internal/singleflight.(*Group).DoChan.func1()
	/opt/hostedtoolcache/go/1.19.8/x64/src/internal/singleflight/singleflight.go:88 +0x36 fp=0xc000070fe0 sp=0xc000070fa8 pc=0x5ee6b6
runtime.goexit()
	/opt/hostedtoolcache/go/1.19.8/x64/src/runtime/asm_amd64.s:1594 +0x1 fp=0xc000070fe8 sp=0xc000070fe0 pc=0x46cee1
created by internal/singleflight.(*Group).DoChan
	/opt/hostedtoolcache/go/1.19.8/x64/src/internal/singleflight/singleflight.go:88 +0x315
```

## Your Environment

```bash
bearer version: 1.3.1
sha: 166c7dbb686dec9f49e1240b210d611dee8c9d79
```

```bash
❯ go version
go version go1.20.2 linux/amd64
```

```bash
❯ uname -a
Linux cactus-ths 6.0.0-kali6-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.0.12-1kali1 (2022-12-19) x86_64 GNU/Linux
```

**Comment by cfabianski** [comment]
Thank you @cosad3s for the report! 
it could be linked to https://github.com/Bearer/bearer/issues/899

We are having a look! 🐛

**Comment by didroe** [comment]
We've just released [1.4.0](https://github.com/Bearer/bearer/releases/tag/v1.4.0) which should fix this issue. Please can you try it out and let us know if it's resolved for you.

**Comment by didroe** [comment]
I'll close this for now as we think it's fixed. Please re-open if you are still experiencing this problem with the latest release
